assets/xml/overlays/ovl_Magic_Dark.xml

```diff
@@ -1,8 +1,11 @@
 <Root>
-    <File Name="ovl_Magic_Dark" BaseOffset="0x80B873D0" RangeStart="0xD10" RangeEnd="0x1740" Segment="128">
-        <!--Symbol Name="gEffUnknown10Tex" Offset="0x04032490"/-->
-        <Texture Name="sDiamondTex" OutName="diamond_texture" Format="i8" Width="32" Height="64" Offset="0xD10"/>
-        <!--<DList Name="sDiamondTexDL" Offset="0x1650"/>
-        <DList Name="sDiamondVertsDL" Offset="0x1700"/>-->
+    <File Name="ovl_Magic_Dark" BaseAddress="0x80B873D0" RangeStart="0xD10" RangeEnd="0x1740" Segment="128">
+        <!--gEffUnknown10Tex should be referenced inside of sNayrusLoveDiamondMaterialDL instead of 0x04032490 -->
+        <Texture Name="sNayrusLoveDiamondTex" OutName="diamond_texture" Format="i8" Width="32" Height="64" Offset="0xD10"/>
+        <Array Name="sNayrusLoveDiamondVtx" Count="20" Offset="0x1510">
+            <Vtx/>
+        </Array>
+        <DList Name="sNayrusLoveDiamondMaterialDL" Offset="0x1650"/>
+        <DList Name="sNayrusLoveModelDL" Offset="0x1700"/>
```

Comment:
```suggestion
        <DList Name="sNayrusLoveDiamondModelDL" Offset="0x1700"/>
```
idk, it's used for the other symbols so why not this one too
(may be better: just don't use "Diamond")